crypto/fieldutils/field.go

```diff
@@ -0,0 +1,45 @@
+package fieldutils
```

Comment:
Some suggestions for better package names:
`random`
`blobgen`
`kzggen`
`kzgrandom`